src/core/lib/security/credentials/credentials.h

```diff
@@ -53,11 +53,21 @@ typedef enum {
   "FakeTransportSecurity"
 #define GRPC_CHANNEL_CREDENTIALS_TYPE_GOOGLE_DEFAULT "GoogleDefault"
 #define GRPC_CREDENTIALS_TYPE_INSECURE "insecure"
+#define GRPC_CREDENTIALS_TYPE_COMPOSITE "Composite"
```

Comment:
My reasoning for having them all in a single place is to just make it easier on someone creating a new type to look over all the existing names so that it doesn't create any confusion. But, given that we are doing pointer comparison instead of string comparison, I don't think it matters one way or the other.